moc_openstack/manifests/ssl/temp_glance_fix.pp

```diff
@@ -0,0 +1,14 @@
+class moc_openstack::ssl::temp_glance_fix (
+  $file   = "/usr/lib/python2.7/site-packages/glanceclient/common/https.py",
+  $source = "puppet:///modules/quickstack/https.py",
+) {
+
```

Comment:
Should we do this or should we pull the updated glanceclient rpm and put it on the localserver?